recipe/meta.yaml

```diff
@@ -62,16 +63,17 @@ requirements:
     - ninja  # [win]
     - {{ compiler('cxx') }}
   host:
+    - llvmdev=5
```

Comment:
is llvm=5 required here (as it is a dependency of clang)

If yes, since clang and llvm must have the same version, maybe specifying `llvmdev={{ clang_version }}`  would be better?

Reply:
Yes it is, I've made `llvmdev` use the `clang_version` variable.

I made `clang_version` a list as `cling` should only depend on the major version of `llvmdev` and it seems jinja2 doesn't have a "split" filter.